docs/sql-ref-syntax-ddl-alter-database.md

```diff
@@ -21,25 +21,47 @@ license: |
 
 ### Description
 
-You can alter metadata associated with a database by setting `DBPROPERTIES`.  The specified property
-values override any existing value with the same property name. Please note that the usage of 
-`SCHEMA` and `DATABASE` are interchangeable and one can be used in place of the other. An error message
-is issued if the database is not found in the system. This command is mostly used to record the metadata
-for a database and may be used for auditing purposes.
+`ALTER DATABASE` statement changes the properties or location of a database. Please note that the usage of
+`DATABASE`, `SCHEMA` and `NAMESPACE` are interchangeable and one can be used in place of the others. An error message
+is issued if the database is not found in the system.
 
-### Syntax
+### ALTER PROPERTIES
+`ALTER DATABASE SET DBPROPERTIES` statement changes the properties associated with a database.
+The specified property values override any existing value with the same property name. 
+This command is mostly used to record the metadata for a database and may be used for auditing purposes.
+
+#### Syntax
 
 ```sql
-ALTER { DATABASE | SCHEMA } database_name
-    SET DBPROPERTIES ( property_name = property_value [ , ... ] )
+ALTER { DATABASE | SCHEMA | NAMESPACE } database_name
+    SET { DBPROPERTIES | PROPERTIES } ( property_name = property_value [ , ... ] )
 ```
 
-### Parameters
+#### Parameters
 
 * **database_name**
 
     Specifies the name of the database to be altered.
 
+### ALTER LOCATION
+`ALTER DATABASE SET LOCATION` statement changes the default parent-directory where new tables will be added 
+for a database. Please note that it does not move the contents of the database's current directory to the newly 
+specified location or change the locations associated with any tables/partitions under the specified database 
+(available in Hive 3.0.0 and later).
```

Comment:
Thanks for the review. I mentioned the Hive version here because the usability of the `ALTER DATABASE SET LOCATION` command depends on the version of the Hive metastore.
I can add the Spark version here (Spark 3.0.0 and later) but I think it's still good to write about Hive.

Reply:
fixed.